spec/models/showback_event/mem_spec.rb

```diff
@@ -33,9 +33,9 @@
     end
 
     it "Calculate MEM_total_mem" do
-      expect(event.MEM_total_mem(1024)).to eq(1024)
-      expect(event.MEM_total_mem(0)).to eq(1024)
-      expect(event.MEM_total_mem(2048)).to eq(2048)
+      expect(event.MEM_total_mem(4096)).to eq(4096)
```

Comment:
Thanks @sergio-ocon this is wrong, this calculate the max_memory used at month and not the actual :( I need change it